src/sidebar/components/annotation-omega.js

```diff
@@ -99,10 +99,23 @@ function AnnotationOmega({
     }
   };
 
+  // Allow saving of annotation by pressing CMD/CTRL-Enter
+  const onKeyDown = event => {
+    if (isEmpty || !isEditing) {
+      return;
+    }
+    if ((event.metaKey || event.ctrlKey) && event.key === 'Enter') {
+      event.stopPropagation();
+      event.preventDefault();
+      onSave();
+    }
+  };
+
   const onToggleReplies = () => setCollapsed(annotation.id, !threadIsCollapsed);
 
   return (
-    <div className="annotation-omega">
+    /* eslint-disable-next-line jsx-a11y/no-static-element-interactions */
```

Comment:
OK, JSX 101 basics here, but I couldn't make this parse in any way if I put it between brackets. I don't think this is a kosher comment. Halp?

Reply:
This got me thinking a little more about where we are adding the onKeyDown event. In most case, I think tis bubbling up from the `<textarea>` so can it just be on that instead? Also, I noticed that this event is triggered from other focusable elements. So, for example, if you write some text, then tab backwards to select one of the markdown buttons, then you can still `CMD+Enter` to save the annotation. Is that what we intend to do? 

![cmd_enter3](https://user-images.githubusercontent.com/3939074/75394463-0847e900-58a5-11ea-9078-c84ed1823c8b.gif)

With respect to the JSX issue, I think the braces `{ }` have to between nodes and cant be at the start of the return where the comment is. But what you can do at the start is just the normal style comment `//` which you have so its perfect!

Reply:
> This got me thinking a little more about where we are adding the onKeyDown event. In most case, I think tis bubbling up from the <textarea> so can it just be on that instead?

For posterity, we talked about this using voices on a call. Ideally, yes, the event handling would be on the proper/sub-component elements. But that would require a bunch more work and refactoring that is risky with the legacy `Annotation` controller still in play.

To clarify: the changes in this PR _directly reimplement_ the existing functionality from `Annotation`.